integrations/openai.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: OpenAI
+---
+
+<Note>
+  We support the OpenAI python library starting at version 1.0.0.
+</Note>
+
+The benefits of this integration is that you can see the OpenAI API calls in a step in the UI, and you can explore them in the prompt playground.
```

Comment:
Also say that it will be persisted in the data layer and visible in Literal if the project is using literal (like token per second metric and such).